packages/corvid-e2e-tests/test/sanity.e2e.js

```diff
@@ -0,0 +1,84 @@
+const tempy = require("tempy");
+const puppeteer = require("puppeteer-core");
+const eventually = require("wix-eventually");
+const { findAvailablePort } = require("./drivers/utils");
+const corvidCliDriverCreator = require("./drivers/cliDriver");
+const localEditorDriverCreator = require("./drivers/localEditorDriver");
+
+const EXIT_CODE_SUCCESS = 0;
+
+const testSites = [
+  {
+    editorUrl:
+      "https://editor.wix.com/html/editor/web/renderer/edit/bdce5f39-c471-4198-9778-c6c8118dbaa8?metaSiteId=d387aea5-55d2-47cf-a348-3d1fbe316245&editorSessionId=bf5f376b-7485-4995-a6c6-d7535d9c7853",
+    description: "blank"
+  },
+  {
+    editorUrl:
+      "https://editor.wix.com/html/editor/web/renderer/edit/356795a0-49ed-4503-b662-2552189be995?metaSiteId=f248b2fc-953b-4c46-85c0-132f851ad6c2&editorSessionId=ebbc915a-dea0-4ad8-8a41-fb2b1fff65ab",
+    description: "inputs"
+  },
+  {
+    editorUrl:
+      "https://editor.wix.com/html/editor/web/renderer/edit/425f3d9a-0d1b-45bd-9ce7-47cdac76d6a4?metaSiteId=ce77f15d-c430-4d6a-bb53-c56cb0f75ffb&editorSessionId=d8b144f4-71c4-42de-a0f4-5e08f4b06674",
+    description: "wix-code"
+  }
+  // {
+  //   editorUrl:
+  //     "https://editor.wix.com/html/editor/web/renderer/edit/4833b200-ce51-4595-b2ac-c7aa33aba5b8?metaSiteId=c29bbfdb-188a-4a08-a96d-4b5bb0e0627f&editorSessionId=c98ba04e-9f37-5b14-9a76-4a47cb07ad51",
+  //   description: "repeaters"
+  // }
+];
+
+const login = async cliDriver => {
+  const remoteDebuggingPort = await findAvailablePort();
+  cliDriver.login({ remoteDebuggingPort });
+  await eventually(async () => {
+    const browser = await puppeteer.connect({
+      browserURL: `http://localhost:${remoteDebuggingPort}`,
+      defaultViewport: { width: 1280, height: 960 }
+    });
+    const [page] = await browser.pages();
+    const localEditorDriver = localEditorDriverCreator(page);
+    await localEditorDriver.login();
+  });
+};
+
+describe("browser sanity", () => {
+  const cliDriver = corvidCliDriverCreator();
+  const initTempDirectory = () => {
+    const cwd = process.cwd();
+    process.chdir(tempy.directory());
+    return () => process.chdir(cwd);
+  };
+
+  beforeEach(async () => {
+    await cliDriver.logout();
+    await login(cliDriver);
+  });
+
+  testSites.forEach(({ editorUrl, description }) =>
+    test(`should clone ${description} site, open it and push without making actual changes`, async () => {
```

Comment:
right now this only checks that cloning and pushing pass without errors, but we don't actually test their "output" (files on the hard-drive)